devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-messaging-codestart/codestart.yml

```diff
@@ -0,0 +1,18 @@
+name: reactive-messaging-codestart
+ref: reactive-messaging-codestart
+tags: extension-codestart
+type: code
+metadata:
+  title: Reactive Messaging codestart
+  description: Use SmallRye Reactive Messaging
+  ## TODO: select the right guide depending on the selected extension ?
+  related-guide-section: https://quarkus.io/guides/kafka
+language:
+  base:
+    dependencies:
+    # TODO: missing HTTP & RabbitMQ connectors ? Not sure to add them as there is no quarkus-smallrye-reactive-messaging extension ?
+      - io.quarkus:quarkus-smallrye-reactive-messaging-kafka
+      - io.quarkus:quarkus-smallrye-reactive-messaging-mqtt
+      - io.quarkus:quarkus-smallrye-reactive-messaging-amqp
```

Comment:
```suggestion
```

You shouldn't add them manually else they will be added automatically in the generated app. We don't want that :)

Reply:
I am guessing that the dependency of the extension is already present in the generated codestart.

Reply:
@ozangunalp a user is going to select one (or more) of the connectors as extensions, the codestart will be binded to all of them (but deduplicted if more than one is selected). Therefore the selected connectors extensions will be in the pom.xml, no need to add them.

Reply:
fixed